extension-manifest-v2/src/classes/EventHandlers.js

```diff
@@ -536,7 +535,7 @@ class EventHandlers {
 	 * @param  {boolean} fromRedirect
 	 * @return {string|boolean}
 	 */
-	static _blockHelper(details, tabId, appId, bugId, requestId, fromRedirect, upgradeInsecure) {
+	static _blockHelper(details, tabId, appId, requestId, fromRedirect, upgradeInsecure) {
```

Comment:
The function is also called at the end onBeforeRequest:

https://github.com/ghostery/ghostery-extension/blob/85b75299f40b97dcb64c853065245d7f1c9ae12f/extension-manifest-v2/src/classes/EventHandlers.js#L337